Core/src/main/java/com/plotsquared/core/util/EntityUtil.java

```diff
@@ -42,27 +42,14 @@ private EntityUtil() {
     }
 
     private static int capNumeral(final @NonNull String flagName) {
-        int i;
-        switch (flagName) {
-            case "mob-cap":
-                i = CAP_MOB;
-                break;
-            case "hostile-cap":
-                i = CAP_MONSTER;
-                break;
-            case "animal-cap":
-                i = CAP_ANIMAL;
-                break;
-            case "vehicle-cap":
-                i = CAP_VEHICLE;
-                break;
-            case "misc-cap":
-                i = CAP_MISC;
-                break;
-            case "entity-cap":
-            default:
-                i = CAP_ENTITY;
-        }
+        int i = switch (flagName) {
```

Comment:
Would be easier to directly return the switch instead of assigning it to a variable with the only goal to immediately return it. Otherwise, not much that can be broken with that change lol